webapp/src/features/adminEditUser/adminEditUserSlice.tsx

```diff
@@ -130,10 +126,14 @@ export const submitForm = createAsyncThunk(
       let submitOrgs = [...selectedOrganizations].map((org) => ({ ...org }))
       submitOrgs.forEach((elm) => delete elm.id)
       const tempData = organizationParser(data, submitOrgs, apiData)
-      dispatch(editUser({ json: tempData }))
-      return false
+      let res = await dispatch(editUser({ json: tempData }))
+      if ((res.payload as any).success) {
+        return { submitAttempt: false, success: true, message: 'User Updated Successfully' }
+      } else {
+        return { submitAttempt: false, success: false, message: (res.payload as any).message }
+      }
     } else {
-      return true
+      return { submitAttempt: true, success: false, message: 'Please fill out all required fields' }
```

Comment:
What is the purpose of this `submitAttempt` variable? Does it refer to whether the user has attempted to submit the form? Renaming the variable to be more descriptive may be helpful here.